projects/angular/src/emphasis/_properties.badges.scss

```
@@ -4,35 +4,44 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 @use '../utils/mixins';
-@use '../utils/variables/variables.global';
+@use '@cds/core/tokens/tokens.scss';
```

Are mixins used any where in the file?
```suggestion
@use '@cds/core/tokens/tokens.scss';
```

---

Yes - we use `mixins.exports('badge.properties')` in the beginning